content/nomos/updates/2024-01-22.md

```diff
@@ -0,0 +1,58 @@
+- [Analyzed](https://www.notion.so/rust-libp2p-QUIC-transport-analysis-6a6acf6f02424f0ab7ba30ecb6b8647b) the rust-libp2p QUIC transport in terms of configuration and implementation details.
+## `testnet`
+
+### `development`
+
+- Removed the `async-trait` from the [node](https://github.com/logos-co/nomos-node/pull/561) as well as [Overwatch](https://github.com/logos-co/Overwatch/pull/33). 
+- Grafana and related services addition to the testnet ([new PR](https://github.com/logos-co/nomos-node/pull/564)).
+- Preparation for demo chat app bot, ability to send data from file, non-interactively ([new PR](https://github.com/logos-co/nomos-node/pull/565)).
+- [Prune Carnot old block PR](https://github.com/logos-co/nomos-node/pull/559) has been completed and is currently in review.
+- Added a [test](https://github.com/logos-co/nomos-node/pull/563) for big blobs dissemination.
+- Limited the number of in-flight requests performed by the chat app to avoid using too many descriptors - more details [here](https://github.com/logos-co/nomos-node/pull/562).
+
+## `cryptarchia`
+
+### `research`
+
+- Updated the wealth concentration analysis with the new [chapter]([https://www.notion.so/Does-Crypsinous-Leader-Election-Function-lead-to-wealth-concentration-in-PoS-b81f07a791b745438443f51f00ac258f?pvs=4#1df422f6cc204cb8b362f41cda260b8b](https://www.notion.so/Does-Crypsinous-Leader-Election-Function-lead-to-wealth-concentration-in-PoS-b81f07a791b745438443f51f00ac258f?pvs=4#1df422f6cc204cb8b362f41cda260b8b) about stake relativization algorithm. The [section](https://www.notion.so/Does-Crypsinous-Leader-Election-Function-lead-to-wealth-concentration-in-PoS-b81f07a791b745438443f51f00ac258f#ca7aa03a7b9541849b71843a46c883c7) about known total stake is also in progress.
+- [Stake relativization](https://www.notion.so/Stake-Relativization-65f5ed5522b64c36b625652023318d88#5ac8c8ead49d4bc7b9ab742bafd91257) specification is complete.
+- Set up an [analytical framework](https://www.overleaf.com/project/656dfacf4929b4a3d6d2ffe5) which will be used to study the impact of using the (biased) total stake estimator on the leader election process. The central object of this framework is the joint probability distribution of two copies of the election process with the same (random) sampling noise, the same stake but different values of the total stake.
+
+### `development`
+
+- No updates, heavily in research.
+## `data availability`
+
+### `research`
+
+- [DA API specification](https://www.notion.so/DA-Api-Specification-83000d9c2f3f45b586cd55c5c195cce5) is written (prone to change due to active discussions)
+- [DA Layer Comparison article](https://www.notion.so/DA-Layer-Comparison-Table-5848811f0af042e2b24c10d3cea9b0a8) is finished: . Protocols were explained in more detail. Tables have been updated. To see the raw updated data on its own refer to this [sheet](https://docs.google.com/spreadsheets/d/1I2hk69hWLVXaATC5048tLcw2qUTjiZORIzL0y90PTLg/edit#gid=0).
```

Comment:
punctuation after finished:

Also, is not really finished, as there are comments open still